sdk/service/backend.go

```diff
@@ -66,7 +65,7 @@ func (s *Backend) querier(request cosmostypes.Request, path []string, req abci.R
 		return s.List(request)
 	case "hash":
 		var createServiceRequest api.CreateServiceRequest
-		if err := proto.Unmarshal(req.Data, &createServiceRequest); err != nil {
+		if err := codec.UnmarshalJSON(req.Data, &createServiceRequest); err != nil {
```

Comment:
I would prefer to use proto for the encoding here as we had issues with the json encoding when using `oneof`. For the service it is fine but if we do that for the process we will have an issue. That was the reason why we used proto to marshal here and at the same time it's more efficient

Reply:
we have to choose one encoding for the data. we cannot use proto and amino and json.
Service is not using any oneof, so what is the issue here?

Reply:
I'm fine to use binary encode from amino, let's just try to avoid json

Reply:
@krhubert can you replace by binary unmarshal? don't forget to replace also the marshal ;)